pkgs/by-name/gi/git-dumper/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
buildPythonApplication rec {
```
using python3.pkgs pins the derivation to a certain python version which is not ideal